src/www/ui/api/Helper/DbHelper.php

```diff
@@ -86,8 +86,10 @@ public function getUploads($userId, $uploadId = null)
 INNER JOIN folderlist ON folderlist.upload_pk = upload.upload_pk
 INNER JOIN folder ON folder.folder_pk = folderlist.parent
 INNER JOIN pfile ON pfile.pfile_pk = upload.pfile_fk
-WHERE upload.user_fk = " . pg_escape_string($userId) . "
+WHERE upload.user_fk = $1
 ORDER BY upload.upload_pk;";
+      $statementName = __METHOD__ . ".getAllUploads";
+      $params = [pg_escape_string($userId)];
```

Comment:
I think the use of `pg_escape_string` is not required here as once a statement is prepared, it's query plan cannot be changed, thus can be attacked by SQL injection.

What are the views of others on this?

Reply:
Instead of using `pg_escape_string`, it will better to use `intval` as `pg_escape_string` will do no good even if a string gets slipped during the call somehow.

Reply:
Ok, @GMishx  - if it's final decision, I can update source branch with this.

If i understood well our previous discussion, source branch should still contain only 1 commit, so I should use `-amend` commit option

Reply:
I think the use of `pg_escape_string` is even harmful here, since it results in double escaping. Just drop it since it is not necessary for prepared statements.

Reply:
Hello @GMishx, @maxhbr - PR updated as You requested (deleted `pg_escape_string`)

Reply:
Approved the changes. Thank you.